Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Sliders/PinchSlider.cs

```diff
@@ -88,6 +88,25 @@ public GameObject TickMarks
             }
         }
 
+        [Tooltip("The gameObject that contains only the thumb Visuals.  This will get rotated to match the slider axis")]
+        [SerializeField]
+        private GameObject thumbVisuals = null;
+        /// <summary>
+        /// Property accessor of thumbVisuals, it contains the desired tick Marks.  This will get rotated to match the slider axis.
```

Comment:
```suggestion
        /// Property accessor of thumbVisuals, it contains the desired tick marks.  This will get rotated to match the slider axis.
```